### PR TITLE
Fix: Profile interests

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- update profile interest click handler
- add snackbar notification to topic handlers
- fix breaking stories

## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: extensions/widgets/trending/src/components/cards/latest-topics.tsx [hunk 1]
```diff
@@ -34,6 +38,15 @@ export type LatestTopicsProps = {
   refetchTagSubscriptions: () => void;
 };
 
+/**
+ * This utility method compares two arrays of tags and returns the difference,
+ * @param arr1 - first array to be compared
+ * @param arr2 - second array to be compared
+ * @returns an array of strings
+ */
+const getTagsDiff = (arr1: string[], arr2: string[]) =>
```
Comment (kenshyx) [inline extensions/widgets/trending/src/components/cards/latest-topics.tsx]: what if the lists are sorted different? this doesn't look it does what it is expected

File: extensions/widgets/trending/src/components/cards/latest-topics.tsx [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
 import { TopicRow } from './topic-row';
-import isEqual from 'lodash/isEqual';
-import difference from 'lodash/difference';
-import pullAll from 'lodash/pullAll';
+
+declare global {
+  interface Set<T> {
+    symmetricDifference: (value: Set<T>) => Set<T>;
+  }
+}
```
Comment (josenriagu) [inline extensions/widgets/trending/src/components/cards/latest-topics.tsx]: @kenshyx is it okay, if we have this global defined here?

File: extensions/widgets/trending/src/components/cards/latest-topics.tsx [hunk 3]
```diff
@@ -34,6 +43,19 @@ export type LatestTopicsProps = {
...
+ * This utility method compares two arrays of tags and returns the difference,
+ * @param arr1 - first array to be compared
+ * @param arr2 - second array to be compared
+ * @returns an array of strings
+ */
+const getTagsDiff = (arr1: string[], arr2: string[]) => {
+  const setA = new Set(arr1);
+  const setB = new Set(arr2);
+
+  return Array.from(setA.symmetricDifference(setB));
```
Comment (josenriagu) [inline extensions/widgets/trending/src/components/cards/latest-topics.tsx:50]: @kenshyx used like so, here

File: extensions/widgets/trending/src/components/cards/latest-topics.tsx (hunk 2 above)
Comment (kenshyx) [reply extensions/widgets/trending/src/components/cards/latest-topics.tsx]: why would you do something like this? this is a type that should come from the typescript lib

Comment (kenshyx) [reply extensions/widgets/trending/src/components/cards/latest-topics.tsx]: merge with next, I've updated the tsconfig for you;
please remove this global overwrite

File: libs/typings/src/ui/ui-events.ts [hunk 4]
```diff
@@ -56,7 +56,6 @@ export const enum NotificationEvents {
  **/
 export const enum NotificationTypes {
   Info = 'info',
-  Alert = 'alert',
```
Comment (kenshyx) [inline libs/typings/src/ui/ui-events.ts:59]: why `Alert` type had to be removed?

Comment (josenriagu) [reply libs/typings/src/ui/ui-events.ts:59]: It was replaced with Info, per the updated design
https://www.figma.com/design/WbIHLctkDhGJjLKLhZcjCf/AKASHA-WORLD-DESIGN-SYSTEM?node-id=13-2121&t=CDxoD4c4899D6Wrj-0

Comment (josenriagu) [reply libs/typings/src/ui/ui-events.ts:59]: We now have only 4 notif types: Info, Caution, Success, Error

**Review by kenshyx — DISMISSED** [review]

**Review by didd — APPROVED** [review]
The PR doesn't address race condition while subscribing and eventually notifying users the topics they subscribed to. I have asked Jose to create a task to handle this.

**Review by kenshyx — APPROVED** [review]

**Pull request merged**